### ISSUE TITLE
Option to disable update checking

### ISSUE THREAD

**Issue by RefineryX** [amir20/dozzle#2694]
**Is your feature request related to a problem? Please describe.**
Dozzle is in an internal network of its own with no access to the internet. The logs are filing with errors relating to the fact it is unable to check for updates.

**Describe the solution you'd like**
Option to disable update checking or limit the log reporting of it?

**Additional context**

```
level=warning msg="error reading releases: Get \"https://api.github.com/repos/amir20/dozzle/releases?per_page=9\": dial tcp: lookup api.github.com on 127.0.0.11:53: read udp 127.0.0.1:55125->127.0.0.11:53: i/o timeout"
```

**Comment by amir20** [comment]
Might be easier to just make the message debug to be honest.

**Comment by RefineryX** [comment]
That makes sense. Making it debug would solve my issue.

**Comment by amir20** [comment]
Fixed with 89babeab

**Comment by amir20** [comment]
Released
